sktime/classification/compose/_pipeline.py

```diff
@@ -161,6 +161,16 @@ def _transformers(self):
     def _transformers(self, value):
         self.transformers_._steps = value
 
+    @property
+    def _steps(self):
+        return self._check_estimators(self.transformers) + [
+            self._coerce_estimator_tuple(self.classifier)
+        ]
+
+    @property
+    def steps_(self):
```

Comment:
question, I do not know the answer - do we need `steps_`?

Reply:
for vizualization - no, but I wanted to follow the convention described here: https://github.com/sktime/sktime/blob/3e1c92bbdceb0332ae6c12026528f650fa0ab4ec/sktime/base/_meta.py#L27-L31